src/Engine/Image.php

```diff
@@ -0,0 +1,76 @@
+<?php
```

Comment:
Maybe this should go to its own namespace, e.g. Image?

Reply:
I think it'd be especially useful if we plan to add more classes to that NS, otherwise, as you said, I don't think it's necessary. One example might be a new Exception class used in Image to assert preconditions on `$this->data`. But again, it's just a minor change and I agree with you that we can leave it here.